packages/supplementary-contracts/script/tokenunlocking/Vest.s.sol

```diff
@@ -0,0 +1,55 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.24;
+
+import "forge-std/src/Script.sol";
+import "forge-std/src/console2.sol";
+
+import "@openzeppelin/contracts/proxy/ERC1967/ERC1967Proxy.sol";
+import "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+
+import "../../contracts/tokenunlocking/TokenUnlocking.sol";
+
+contract VestTokenUnlocking is Script {
+    using stdJson for string;
+
+    struct VestingItem {
+        bytes32 name; // Conversion from json "string" to bytes32 will take place in foundry,
+            // cannot use string here, as json parser cannot interpret string from json, everything
+            // is bytes-chunks. It is more of informational to script executor anyways.
+        address recipient;
+        address proxy;
+        uint256 vestAmount;
+    }
+
+    uint256 public PRIVATE_KEY = vm.envUint("PRIVATE_KEY"); // owner
```

Comment:
We dont need this PRIVATE_KEY in this implementation then. Because we use the `msg.sender `(`line 39`) we need to call the script in anyways with the `--private-key` flag with `forge script`.